bot/cogs/sync/cog.py

```diff
@@ -30,54 +30,68 @@ def __init__(self, bot):
         self.bot = bot
 
     async def on_ready(self):
+        """Syncs the roles/users of the guild with the database."""
         guild = self.bot.get_guild(self.SYNC_SERVER_ID)
         if guild is not None:
             for syncer in self.ON_READY_SYNCERS:
                 syncer_name = syncer.__name__[5:]  # drop off `sync_`
                 log.info("Starting `%s` syncer.", syncer_name)
-                total_created, total_updated = await syncer(self.bot, guild)
-                log.info(
-                    "`%s` syncer finished, created `%d`, updated `%d`.",
-                    syncer_name, total_created, total_updated
-                )
+                total_created, total_updated, total_deleted = await syncer(self.bot, guild)
+                if total_deleted is None:
+                    log.info(
+                        "`%s` syncer finished, created `%d`, updated `%d`.",
+                        syncer_name, total_created, total_updated
+                    )
+                else:
+                    log.info(
+                        "`%s` syncer finished, created `%d`, updated `%d`, `%d` deleted.",
+                        syncer_name, total_created, total_updated, total_deleted
+                    )
 
     async def on_guild_role_create(self, role: Role):
+        """Adds newly create role to the database table over the API."""
         await self.bot.api_client.post(
             'bot/roles',
             json={
                 'colour': role.colour.value,
                 'id': role.id,
                 'name': role.name,
-                'permissions': role.permissions.value
+                'permissions': role.permissions.value,
+                'position': role.position,
             }
         )
 
     async def on_guild_role_delete(self, role: Role):
-        log.warning(
-            (
-                "Attempted to delete role `%s` (`%d`), but role deletion "
-                "is currently not implementeed."
-            ),
-            role.name, role.id
-        )
+        """Deletes role from the database when it's deleted from the guild."""
+        await self.bot.api_client.delete('bot/roles/' + str(role.id))
```

Comment:
This is the prevalent style in this cog package. I can change all entries or keep the ones I've added consistent.

I figured it was better to keep the got blame mostly intact, since a single `+` operation is not something that's unclear or in any way less performant. 

I can change all the occurences, though. I'm pretty neutral on it.

Reply:
I've changed all the occurrences to f-strings.

Reply:
I've changed all the occurrences to f-strings.